server/modules/DeploymentLogsStreamer.js

```diff
@@ -33,39 +33,10 @@ module.exports = function DeploymentLogsStreamer() {
     return logStreams;
   }
 
-  function getDeploymentHistory(deploymentId, accountName) {
-    var query = {
-      name: 'GetDynamoResource',
-      resource: 'deployments/history',
-      accountName: accountName,
-      key: deploymentId,
-    };
-
-    return sender.sendQuery({ query: query });
-  }
-
   function flushLogStream(logStream) {
     return co(function* () {
-      var deploymentHistory = yield getDeploymentHistory(
-        logStream.deploymentId, logStream.accountName
-      );
-
-      var executionLog = deploymentHistory.Value.ExecutionLog;
-      var executionLogEntries = executionLog ? executionLog.split('\n') : [];
-
-      executionLogEntries = executionLogEntries.concat(logStream.logs);
-
-      var command = {
-        name: 'UpdateDynamoResource',
-        resource: 'deployments/history',
-        accountName: logStream.accountName,
-        key: logStream.deploymentId,
-        item: {
-          'Value.ExecutionLog': executionLogEntries.join('\n'),
-        },
-      };
-
-      yield sender.sendCommand({ command: command, user: systemUser });
+      let deployment = yield Deployment.getById(logStream.deploymentId)
```

Comment:
Nice. This will find the deployment if it is in `ConfigDeploymentExecutionStatus` or `ConfigCompletedDeployments`